src/Lykke.Job.TransactionHandler/Modules/JobModule.cs

```diff
@@ -285,6 +285,10 @@ private void BindRepositories(ContainerBuilder builder)
 
             builder.RegisterInstance<IClientCommentsRepository>(
                 new ClientCommentsRepository(AzureTableStorage<ClientCommentEntity>.Create(_dbSettingsManager.ConnectionString(x => x.ClientPersonalInfoConnString), "ClientComments", _log)));
+
+            builder.RegisterInstance<IEthereumCashinAggregateRepository>(
```

Comment:
Make it singletone, since in some Lykke.Azure versions transient repository will lead to some extra calls to the storage

Reply:
builder.RegisterInstance
RegisterInstance
Register... Instance...